build.rs

```diff
@@ -25,10 +26,17 @@ fn main() {
     nssm_dest.push(".."); 
     nssm_dest.push(".."); 
     nssm_dest.push("nssm");
+
+    let mut logger_dest = PathBuf::from(out_dir.clone());
+    logger_dest.push(".."); 
+    logger_dest.push(".."); 
+    logger_dest.push(".."); 
+    logger_dest.push("Logs");
```

Comment:
why we need to move log file to the build?

Reply:
i fix this now